src/test/java/generics/BoxTest.java

```diff
@@ -0,0 +1,98 @@
+package generics;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class BoxTest {
+
+    @Test
+    void getList() {
+        Box<Apple> box = new Box();
+        Apple apple1 = new Apple();
+        Apple apple2 = new Apple();
+        box.add(apple1);
+        box.add(apple2);
+        List<Apple> expected = new LinkedList();
+        expected.add(apple1);
+        expected.add(apple2);
+        assertEquals(expected, box.getItems());
+    }
+
+    @Test
+    void add() {
+        Box<Apple> box = new Box();
+        Apple apple1 = new Apple();
+        Apple apple2 = new Apple();
+        box.add(apple1);
+        Apple result = box.add(apple2);
+        assertEquals(apple2, result);
+    }
+
+    @Test
+    void testAdd() {
+        Box<Apple> box = new Box();
+        Apple apple1 = new Apple();
+        Apple apple2 = new Apple();
+        List<Apple> apples = new LinkedList();
+        apples.add(apple1);
+        apples.add(apple2);
+        List<Apple> result = box.add(apples);
+        assertEquals(apples, result);
+    }
+
+    @Test
+    void getWeight() {
+        Box<Apple> box = new Box();
+        Apple apple1 = new Apple();
+        Apple apple2 = new Apple();
+        Apple apple3 = new Apple();
+        box.add(apple1);
+        box.add(apple2);
+        box.add(apple3);
+        assertEquals(3, box.getWeight());
+    }
+
+    @Test
+    void compare() {
+        Box<Apple> box1 = new Box();
```

Comment:
```suggestion
        Box<Apple> box1 = new Box<>();
```
 IDEA хіба не підказує?

Reply:
Ні(